__tests__/upgradeConfig.test.js

```diff
@@ -71,3 +71,14 @@ test('does not another flow preset if already present and hasFlow option passed'
   expect(upgradeConfig(config, { hasFlow: true })).toMatchSnapshot();
 });
 
+test('does not another flow preset if already present in an array and hasFlow option passed', () => {
```

Comment:
noticed this mid-refactor, not sure if this is a case worth handling -- but i figured it couldn't hurt to be safe

Reply:
minor grammar fix: 
```diff
- does not another
+ does not add another
```

Reply:
thanks! nice catch